account/delete-account.md

```diff
@@ -10,6 +10,10 @@ keywords:
 - delete
 ---
 
-In the Clever Cloud Web Console, select **Profile** in the bottom left menu, then select **Delete my account** in the top menu of the freshly opened page.
+To delete your account :
+1) Enter un the Clever Cloud Web Console.
```

Comment:
I think it would be better to use classic bullet points (using `-`), or at least the markdown numbered list (like `1.`, `2.` and so on), we do not use `1)` notation elsewhere.

Also, this line might be better with `Open the Clever Cloud Web Console`

Reply:
Because the order of items are important I suggest keeping the numbered list to respect technical writing guidance. But the 1. notation can be used if it's like that in another part of the documentation.

Open it's also better than enter yes.